Gems/WhiteBox/Code/Source/EditorWhiteBoxComponent.cpp

```diff
@@ -720,7 +720,6 @@ namespace WhiteBox
         // must have at least one triangle
         if (m_faces->empty())
```

Comment:
This isn't directly connected but I ran into it when looking at the intersection code, we really should not be writing to `distance` unless there was a successful hit so I wanted to fix this quickly when I saw it